tests/middleware_tests/test_redirect_handler.py

```diff
@@ -71,9 +71,3 @@ def test_is_not_https_redirect(mock_redirect_handler):
     url = httpx.URL("http://example.com")
     location = httpx.URL("https://www.example.com")
     assert not mock_redirect_handler.is_https_redirect(url, location)
-
-
-def test_is_not_https_redirect_if_not_default_ports(mock_redirect_handler):
```

Comment:
this unit test is sill valuable

Reply:
How so? Didn't we just say port comparisons are not necessary when checking https redirection.

Reply:
nevermind, I misread one of the other tests.